airflow/api_connexion/endpoints/health_endpoint.py

```diff
@@ -28,7 +29,9 @@ def get_health() -> APIResponse:
     """Return the health of the airflow scheduler and metadatabase."""
     metadatabase_status = HEALTHY
     latest_scheduler_heartbeat = None
+    latest_triggerer_heartbeat = None
     scheduler_status = UNHEALTHY
+    triggerer_status = None
```

Comment:
I feel this should match the scheduler status logic and default to UNHEALTHY.

Reply:
That'd mean that installations/deployments that don't include a `triggerer` would always report an unhealthy status right? Feels like that could be a little misleading

Reply:
Maybe a better approach would be to set `None` when there is not a `triggerer_job`? As the current logic is implemented, the triggerer status would be None when `most_recent_job` fails, which seems wrong to me.

Reply:
@uranusjr I see what you mean. Updated here - https://github.com/apache/airflow/pull/27755/commits/e71a3ad9be883e12e9f792735aae452c11932965